graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/SubscriptionAutoConfiguration.kt

```diff
@@ -39,6 +38,16 @@ import org.springframework.web.reactive.socket.server.support.WebSocketHandlerAd
 @ConditionalOnBean(Subscription::class)
 class SubscriptionAutoConfiguration {
 
+    /**
+     * This value is needed so that this url handler is run without a drastically different order
+     * to the graphql routes in [RoutesConfiguration]. If we use [org.springframework.core.Ordered] to set as extreme
+     * high or low, then the requests are not handled properly.
+     *
+     * Hopefully we can eventually move the url handler to the same router DSL.
+     * https://github.com/spring-projects/spring-framework/issues/19476
+     */
+    private val urlHandlerOrder = 0
```

Comment:
should this be `const`?

Reply:
const can only be used in a companion object, do we want to move it to an object?

Reply:
instead of companion object i'd move it to top level field

Reply:
Done